Playbooks/AS-Slack-Integration/azuredeploy.json

```diff
@@ -0,0 +1,505 @@
+                                    {
+                                        "equals": [
+                                            "@length(triggerBody()?['object']?['properties']?['relatedEntities'])",
+                                            0
+                                        ]
+                                    }
+                                ]
+                            },
+                            "type": "If"
+                        },
+                        "HTTP_-_Post_Slack_Message": {
+                            "runAfter": {
+                                "Condition_-_Check_if_Entities_Exist": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "Http",
+                            "inputs": {
+                                "body": {
+                                    "blocks": [
+                                        {
+                                            "text": {
+                                                "text": "*@{triggerBody()?['object']?['properties']?['title']}*\n> @{variables('Description')} ",
+                                                "type": "mrkdwn"
+                                            },
+                                            "type": "section"
+                                        },
+                                        {
+                                            "accessory": {
+                                                "alt_text": "severity icon",
+                                                "image_url": "@variables('Severity Icon')",
+                                                "type": "image"
+                                            },
+                                            "text": {
+                                                "text": "*Severity*\n>@{triggerBody()?['object']?['properties']?['severity']}\n\n*Entities*\n@{variables('Entities')}",
+                                                "type": "mrkdwn"
+                                            },
+                                            "type": "section"
+                                        },
+                                        {
+                                            "text": {
+                                                "text": "<@{triggerBody()?['object']?['properties']?['incidentUrl']}|Open incident on Azure Portal>",
+                                                "type": "mrkdwn"
+                                            },
+                                            "type": "section"
+                                        },
+                                        {
+                                            "type": "divider"
+                                        }
+                                    ],
+                                    "channel": "@variables('ChannelId')"

```

Comment:
Refer to parameter **ChannelId** here. No need for a separate variable.

Reply:
This is resolved